src/main/features/response/views/counter-claim.njk

```diff
@@ -1,47 +1,40 @@
 {% extends "layout.njk" %}
 {% from "form.njk" import radioGroup, saveAndContinueButton, errorSummary %}
 {% from "support.njk" import supportWidget %}
+{% from "feesTable.njk" import feesTable %}
 
 {% set title = t('Money claim') %}
 
-{% set heading = t('Next steps') %}
+{% set heading = t('How to complete and send your response') %}
 
 {% block content %}
   <div class="grid-row">
     <div class="column-two-thirds">
       <div class="panel-indent">
         <div class="panel panel-border-wide">
-          <p>{{ t('Complete and email the defence and counterclaim form by %s',
-            { postProcess: 'sprintf', sprintf: [claim.responseDeadline | date]}) }}</p>
-          <p>{{ t('To make a counter claim you will have to pay a fee. Court staff can tell you how much you have to pay.') }}</p>
+          <p>{{ t('Complete and email the defence and counterclaim form by <span class="bold-small">{{ responseDeadline }}', { responseDeadline: claim.responseDeadline | date  }) | safe }} </span></p>
         </div>
       </div>
 
+      <p>{{ t('You’ll need to pay a court fee if you make a counterclaim.') }}</p>
+
+      {{ feesTable('View claim fees', rows, showHearingFee = true ) }}
```

Comment:
shouldn't this one hide the hearing fee?

Reply:
you're right, removed. although it looks like that page has now changed again :/